ci/delete_current_device_c8y.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+"""
+Delete device from Cumulocity with a given name.
+
+For example:
+
+    python3 ci/delete_current_device.py --tenant t493319102 --user octocat
+      --device devraspberrypi --url thin-edge-io.eu-latest.cumulocity.com
+
+"""
+
+import argparse
+import os
+import sys
+from c8y_api import CumulocityApi
+
+
+def delete_object(c8y, obj):
+    """Delete object from inventory"""
+    try:
+        c8y.inventory.get(obj).delete()
+    except KeyError:
+        print(f"Object with {obj} not there")
+    print(f"Deleted object with ID {obj}")
+
+
+def delete_device(c8y, name, verbose):
+    """Delete the current device"""
+    for dev in c8y.device_inventory.get_all():
```

Comment:
`get_all` supports a `name` parameter. That didn't work?

Reply:
Thanks for the hint, I kind of missed that feature. That way we can avoid the loop.